flashlight/app/asr/CMakeLists.txt

```diff
@@ -50,7 +51,7 @@ install(TARGETS fl_asr_decode RUNTIME DESTINATION ${FL_INSTALL_BIN_DIR})
 add_executable(fl_asr_tutorial_inference_ctc ${CMAKE_CURRENT_LIST_DIR}/tutorial/InferenceCTC.cpp)
 add_executable(fl_asr_tutorial_finetune_ctc ${CMAKE_CURRENT_LIST_DIR}/tutorial/FinetuneCTC.cpp)
 
-target_link_libraries(fl_asr_tutorial_inference_ctc fl_pkg_speech fl_pkg_runtime ${CMAKE_DL_LIBS})
+target_link_libraries(fl_asr_tutorial_inference_ctc fl_pkg_speech fl_pkg_runtime fl_lib_text ${CMAKE_DL_LIBS})
```

Comment:
This should be transitively linked since `fl_pkg_speech` requires it — does it fail to link without it?

Reply:
Nope, you are correct. We don't need to have fl_lib_text.